FutureFeatures.md

```diff
@@ -273,6 +273,7 @@ use cases:
   * `i32.abs_s`: signed absolute value (traps on `INT32_MIN`)
   * `i32.bswap`: sign-agnostic reverse bytes (endian conversion)
   * `i32.bswap16`: sign-agnostic, `bswap16(x)` is `((x>>8)&255)|((x&255)<<8)`
+  * `i64.bswap`: sign-agnostic reverse bytes (endian conversion, for 64bit host)
```

Comment:
The other opcodes (`min_s` etc.) are also listed only in `i32` form here; it seems we should either list all of them as both `i32` and `i64`, or just have the `i32` versions and say "and corresponding `i64` versions").

Reply:
Fixes with full i64 alternative